server/routes/juror-management/juror-record/juror-record.controller.js

```diff
@@ -228,7 +237,7 @@ const { defaultExpensesDAO, jurorBankDetailsDAO } = require('../../../objects/ex
         require('request-promise'),
         app,
         req.session.authToken,
-        'overview',
+        'detail',
```

Comment:
do we not use the `overview` endpoint anymore?

Reply:
On the overview tab, we can request the /detail endpoint which contains all the dat from /overview + the data that we need in order to decide whether or not a police check can be run